### ISSUE TITLE
In a multi-activity app, the plugin doesn't reattach to the first activity

### ISSUE THREAD

**Issue by MElkady** [pichillilorenzo/flutter_inappwebview#732]
- [x] I have read the [README](https://github.com/pichillilorenzo/flutter_inappwebview/blob/master/README.md)
- [x] I have already searched for the same problem
- [x] I have read and done the setup for Android: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for Android](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android)
- [x] I have read and done the setup for iOS: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-ios)
- [x] I have ran the sample app and it does not work there

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |      1.22.6         |
| Plugin version       |         4.0.0+4      |
| Android version      |        10       |
| iOS version          |          Not relevant     |
| Xcode version        |       Not relevant        |

Device information: Samsung Galaxy S8

## Description
In an application that consists of multiple flutter activities. The plugin attaches only to the topmost activity. After that activity finishes, `Shared.activity` will become `null` & none of the other activities can use the plugin

## Steps to reproduce

1. Create a flutter application with flutter_inappwebview plugin
2. Then add an additional FlutterActivity (I do that to start a special routine for some use cases)
3. Then build and run the app
4. Then open the main activity (then `Shared.activity` will point to the main activity)
5. Then open the 2nd activity (then `Shared.activity` will point to the 2nd activity)
6. Return to the main activity & try to open the wevbiew
7. Notice that it doesn't work & a null pointer exception is thrown

## Stacktrace/Logcat   
```
[ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: PlatformException(error, Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference, null, java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
        at com.pichillilorenzo.flutter_inappwebview.ChromeSafariBrowserManager.open(ChromeSafariBrowserManager.java:66)
        at com.pichillilorenzo.flutter_inappwebview.ChromeSafariBrowserManager.onMethodCall(ChromeSafariBrowserManager.java:49)
        at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
        at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
        at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:326)
        at android.os.Looper.loop(Looper.java:181)
        at android.app.ActivityThread.main(ActivityThread.java:7050)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

**Comment by pichillilorenzo** [comment]
Did you try the latest version of the plugin and the latest of flutter?? Version `4` is not supported anymore.

However, the plugin already implements the `ActivityAware` flutter interface and support the `onAttachedToActivity` and `onReattachedToActivityForConfigChanges` events (just check the Android `InAppWebViewFlutterPlugin.java` class).
So, I don't know if you need to do something more specific when you move from one Android FlutterActivity to another, but this should already work (I think). It could be also a Flutter issue.
Also, could you create a simple project with that scenario? Thanks.

**Comment by pichillilorenzo** [comment]
I was able to reproduce the error! The fix should come with the next release!

**Comment by pichillilorenzo** [comment]
Fixed with new release `5.3.0`!
